frontend/src/utils/index.js

```diff
@@ -459,6 +459,7 @@ export function transformHtml (html, transformToExternalLinks = true) {
     if (transformToExternalLinks) {
       linkElement.classList.add('text-decoration-none')
       linkElement.setAttribute('target', '_blank')
+      linkElement.setAttribute('rel', 'noopener')
```

Comment:
Why not?
```suggestion
      linkElement.setAttribute('rel', 'noreferrer')
```

Reply:
do not change (now) as we use `noopener` everywhere